codespell_lib/data/dictionary.txt

```diff
@@ -5406,8 +5407,9 @@ composit->composite
 compount->compound
 comppatible->compatible
 comprable->comparable
+compre->compare
```

Comment:
Suggest compère too

Reply:
First time I've seen that word and slightly unusual in code, but OK

Reply:
Done.